src/compiler/types.ts

```diff
@@ -3907,6 +3918,8 @@ namespace ts {
         resolvedModule: ResolvedModuleFull | undefined;
         /* @internal */
         failedLookupLocations: string[];
+        /*@internal*/
+        isInvalidated?: boolean;

```

Comment:
Actually, it looks like you did do that, but forgot to remove the `isInvalidated` here.